main.go

```diff
@@ -100,25 +102,35 @@ func readFlags() {
 	} else if flags.batchSize > 1 {
 		log.Printf("Using fixed batch size of %d\n", flags.batchSize)
 	}
+
+	// check if template exist at the provided path
+	// TODO: add validation for correctness of template
+	if flags.output != "karate" {
```

Comment:
If we are going to support just karate as a built-in, then this should be okay. If we look to support more, though, we'll probably want to hoist those template names into a global variable so we can use it here and in the flag description.

Reply:
I keep that open to you and @wtait1 to decide the default supported template. My idea is to go ahead with karate as default, ship it with package and for rest, user can pass path to template.

Reply:
I say we should keep both for now - this task is more about changing how templates are provided to the test generation process. See my longer comment below for more detail on how I think we should ship the default templates